server/src/database/boards.go

```diff
@@ -167,3 +168,17 @@ func (d *Database) GetBoard(id uuid.UUID) (Board, error) {
 	err := d.db.NewSelect().Model(&board).Where("id = ?", id).Scan(context.Background())
 	return board, err
 }
+
+func (d *Database) GetUserBoards(userID uuid.UUID) ([]Board, error) {
+	var boards []Board
+	err := d.db.NewSelect().
+		TableExpr("boards AS b").
+		ColumnExpr("b.*").
+		Join("INNER JOIN board_sessions AS s ON s.board = b.id").
+		Where("s.user = ?", userID).
+		Scan(context.Background(), &boards)
```

Comment:
I think it would be nice if fetching the user sessions would already include all the required data in #3376, like the amount of columns and participants to avoid multiple BE calls, what do you think?

Reply:
Sounds like a good idea, take a closer look on how to implement it.

Reply:
> I think it would be nice if fetching the user sessions would already include all the required data in #3376, like the amount of columns and participants to avoid multiple BE calls, what do you think?

Would you rather have the column and participant data already counted or the full individual objects and count them in the frontend?

Reply:
By the look of the current design, we need the names of the columns as well, for participants I'm not sure. Maybe @Kraft16 knows more?